[ADD] accounting: basiq bank synchronization (#17747)
odoo/documentation: content/applications/finance/accounting/bank/bank_synchronization/basiq.rst

Reviewer: Maybe link the privacy page about Basiq here: https://production.odoofin.com/privacy#basiq

We have important info as well

> =====

Reviewer 2: Not sure if we have to do it but maybe we need to talk about the consent management ? 

Also maybe we should specify that it's only available for Australian users ? Sadly, Basiq is only providing Web Scrapping for banks in New Zealand. And so until they implement the Open Banking for NZ, we won't have access to it  

Do we need to add a little tip section for the additional requirement like abn and stuff ?

Author: Hi both, many thanks for the suggestions! I addressed your comments in the latest commit. I have one question though, regarding the 'ABN and stuff':

I had already added this line #. Fill out the additional information when prompted, and :guilabel:`Continue`. to tell users they must fill out these fields (including ABN), but is there something specific they should know?

Reviewer: Hello! I don't know if it's relevant in the documentation but we are checking the ABN with the administration to be sure that is a valid one. So we could have an error if it's not a right one

Author: > Hello! I don't know if it's relevant in the documentation but we are checking the ABN with the administration to be sure that is a valid one. So we could have an error if it's not a right one

If the error message is clear and explicit that the ABN is invalid, then I don't think it is necessary to have a note for the documentation. The user will know from the error message where the problem lies and administrative issues are outside Odoo's (documentation) and scope :)

Many thanks for your time and feedback on this!

Basiq
=====

> Basiq is a paid third-party service that enables companies and professionals to connect multiple

Reviewer: It's a free one for our customers 😄

bank accounts to a single platform. It provides a unified view of all transactions within a single
interface. When integrated with Odoo, it automatically synchronizes bank transactions directly into
its database.

.. note::

>    Basiq bank synchronization service is free for Odoo users.

Reviewer: Oh you put it there, maybe line 5 is a bit confusing then?

=====
Basiq
=====

> Basiq is a third-party service that enables companies and professionals to connect multiple bank

Reviewer 3: Add external link on 'Basiq' like we do on the Ponto page?

accounts to a single platform. It provides a unified view of all transactions within a single
interface. When integrated with Odoo, it automatically synchronizes bank transactions directly into
its database.

.. note::

>    - Basiq bank synchronization service is free for Odoo users.

Reviewer 3: ```suggestion
   - Basiq bank synchronization is free for Odoo users.
```

>    - Basiq bank synchronization is currently only available in Australia.

Reviewer 3: Maybe, to avoid the duplication at the start of the sentence, we could get away with:

```suggestion
   - This service is currently only available in Australia.
```

.. seealso::
  - :doc:`../bank_synchronization`
  - `Financial institutions available on Basiq <https://www.basiq.io/resources/supported-insitutions.html>`_

.. _bank-synchronization/basiq/configuration:

Configuration
=============

Basiq account creation
----------------------

To create a Basiq account, go to the `Basiq website <https://dashboard.basiq.io/register>`_ and

> follow the registration instructions.

Reviewer 3: Is there a reason you're being so brief here, compared with how we do for Ponto?

Author: The process is very straightforward (email, name, etc...)  and nothing specific in relation to Odoo needs to be done

.. _bank-synchronization/basiq/payments:

> Business banking bata sharing

Reviewer 3: ```suggestion
Business banking data sharing
```

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Under Australia's Consumer Data Right (CDR) regime, sharing business banking data (for entities

> like Sole Traders, Partnerships, Companies, and Trusts) requires an explicit **opt-in**.

Reviewer 3: Do you think we _need_ to capitalise Sole Traders, Partnerships, etc?
If we don't, I wouldn't.

Consequently, setting up a business account with Basiq may require additional configuration steps
compared to a standard personal account. For detailed setup instructions, refer to Basiq's
`online documentation <https://api.basiq.io/docs/connecting-business-accounts-via-cdr>`_.

.. seealso::

>    - `CDR Compliance <https://api.basiq.io/docs/cdr-compliance>`_

Reviewer 3: ```suggestion
   - `CDR compliance <https://api.basiq.io/docs/cdr-compliance>`_
```

I know that's how the page is capitalized, but don't think we need to follow that.

   - To see how Odoo manages your financial data and how to access, update, or delete it, read the

>      `Odoo finance privacy policy page <https://production.odoofin.com/privacy#basiq>`_.

Reviewer 3: This feels a little heavy in a 'See also' ... though I realise I don't have it very clear in my mind _when_ we use a 'See also' rather than a note.
But I'm thinking that maybe this point would be better in a note, positioned before the 'See also' which would then just have the external link about CDR compliance.

.. _bank-synchronization/basiq/odoo-connection:

Connection with Odoo
--------------------

When :ref:`connecting a bank to Odoo <accounting/bank-synchronization/first-synchronization>` using
Basiq as the third-party provider, follow these steps:

#. When connecting to the desired bank, make sure Basiq is selected as the third-party
   :guilabel:`Provider`.
#. Select the :guilabel:`Type of account` and click :guilabel:`Connect`.
#. Fill out the additional information when prompted, and :guilabel:`Continue`.
#. Agree to share data by checking you declare using this service for business purposes, and then

>    :guilabel:`Approve`.

Reviewer 3: Something a bit odd about '.. checking you declare using ...'. Could this work?

```suggestion
#. Agree to share data by declaring that you are using this service for business purposes, and then
   :guilabel:`Approve`.
```

> #. In the pop-up window, login to Basiq using your **Basiq credentials**. Upon a successful

Reviewer 3: ```suggestion
#. In the pop-up window, log in to Basiq using your **Basiq credentials**. Upon a successful
```

(since it's the verb 'to log in', rather than an adjective, e.g., 'a login screen')

>    connection, you will be redirected to the **Odoo Accounting** dashboard.

Reviewer 3: Not sure we need to bold 'Odoo Accounting ...'

Or actually, maybe:

```suggestion
   connection, you will be redirected to the Odoo Accounting :guilabel:`Dashboard`.
```

#. Select the specific bank account to connect to, and click :guilabel:`Connect Bank`.
#. Access your **Bank Reconciliation** dashboard using your **bank** journal to verify if the

>    synchronization was successful.

Reviewer 3: I can't test any of this so just going to have to trust you 😉 

<img width="671" height="235" alt="Image" src="https://github.com/user-attachments/assets/1420715d-81b7-4050-8bd0-911bd8612ef0" />

>    synchronization was successful.

Reviewer 3: Is this ...

<img width="571" height="366" alt="Image" src="https://github.com/user-attachments/assets/3e4a10d2-1887-42c7-a7b1-cac86b64c163" />

... where I should be finding my Bank Reconciliation dashboard?

```suggestion
#. Access your **Bank Reconciliation** dashboard using your **Bank** journal to verify if the
   synchronization was successful.
```

Author: I slightly changed the wording to make it closer to actual use

.. tip::

>    Repeat the process for all banking institutions and accounts for which you need synchronization.

Reviewer 3: Suggestion:

```suggestion
   Repeat the process for all banking institutions and accounts that need to be synchronized with your database.
```

=====
Basiq
=====

`Basiq <https://www.basiq.io/home.html>`_ is a third-party service that enables companies and
professionals to connect multiple bank accounts to a single platform. It provides a unified view of
all transactions within a single interface. When integrated with Odoo, it automatically synchronizes
bank transactions directly into its database.

.. note::
   - Basiq bank synchronization is free for Odoo users.
   - This service is currently only available in Australia.

.. seealso::
  - :doc:`../bank_synchronization`
  - `Financial institutions available on Basiq <https://www.basiq.io/resources/supported-insitutions.html>`_

.. _bank-synchronization/basiq/configuration:

Configuration
=============

Basiq account creation
----------------------

To create a Basiq account, go to the `Basiq website <https://dashboard.basiq.io/register>`_ and

> follow the registration instructions.

Reviewer: I just noticed it right now but we don't have to create a Basiq account 😬 It's for the support only 😅

Author: I'll make another PR to fix that later

=====
Basiq
=====

Basiq is a paid third-party service that enables companies and professionals to connect multiple
bank accounts to a single platform. It provides a unified view of all transactions within a single
interface. When integrated with Odoo, it automatically synchronizes bank transactions directly into
its database.

.. note::

>    Using Basiq is subject to applicable fees.

Reviewer: Basiq is free for our users

.. seealso::
   - :doc:`../bank_synchronization`
   - `Financial institutions connected to Basiq <https://www.basiq.io/resources/supported-insitutions.html>`_

Configuration
=============

To connect a bank with Odoo using Basiq as the third-party provider, follow these steps (in Odoo):

#. Set up your email address: Click the profile icon in the upper-right corner, select
   :guilabel:`My profile`, and fill in the :guilabel:`Email` field.

> #. :ref:`Create a Ponto account <accounting/bank-synchronization/ponto/ponto-account-creation>` if

Reviewer: You mean Basiq, right?

   needed.
#. :ref:`Connect Odoo to your bank <accounting/bank-synchronization/ponto/odoo-connection>`.

.. _accounting/bank-synchronization/basiq/ponto-account-creation:

Basiq account creation
----------------------

#. Access the `Basiq website <https://dashboard.basiq.io/register>`_ and follow the instructions to
   create a new account.

   .. note::
      Use the same email address as the one :ref:`used for your Odoo profile
      <accounting/bank-synchronization/ponto/configuration>`.

#. After logging in, create an :guilabel:`Organization` and provide the necessary contact details.

> #. Give consent to Ponto to access the bank account data.

Reviewer: Basiq right?

#. Fill out the additional information when prompted, and :guilabel:`Continue`.
#. Agree to share data by checking you declare using this service for business purposes, and then
   :guilabel:`Approve`.
#. Select the specific bank account to connect.

.. tip::
   Select all accounts that need access and synchronization, including those from other banking
   institutions.

.. seealso::
   :ref:`Update synchronization credentials <accounting/bank-synchronization/update-credentials>`

.. _accounting/bank-synchronization/basiq/payments:

Vendor payments with Basiq
==========================

.. important::
   Before managing vendor payments with Basiq, make sure the following steps are completed:

   - :ref:`SEPA Credit Transfer is activated (SCT) <accounting/pay_sepa/activate-sepa>`.
   - :ref:`The SEPA Credit Transfer payment method has been added to the bank journal
     <accounting/pay_sepa/activate-sepa-bank-journal>`.
   - :doc:`The vendor's bank account has been marked as trusted <../../payments/trusted_accounts>`.
   - The :guilabel:`Account Online Payment` (`account_online_payment`) module is :ref:`installed
     <general/install>`.

> 

Reviewer: We don't have payment with basiq, SEPA is not available in Australia
